assets/onboarding/navigation/index.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect, useMemo } from '@wordpress/element';
+import { Stepper } from '@woocommerce/components';
+import { get, uniq } from 'lodash';
+
+import { useQueryStringRouter } from '../query-string-router';
+
+/**
+ * Merge the navigation state into the routes.
+ * Add isComplete and onClick - when visited.
+ *
+ * @param {Array}    routes        Routes list.
+ * @param {string[]} visitedRoutes Key of the visited routes.
+ * @param {Function} updateRoute   Function that update the route.
+ */
+const getRoutesWithNavigationState = ( routes, visitedRoutes, updateRoute ) =>
```

Comment:
This entire function will likely be removed when we integrate the data persistence.
It's simulating the complete steps while the user is navigating, but when the user refreshes the page, we lose it.